modules/zrtp/zrtp.c

```diff
@@ -198,15 +244,74 @@ static bool udp_helper_recv(struct sa *src, struct mbuf *mb, void *arg)
 }
 
 
+static void sig_hash_encode(struct zrtp_stream_t *stream,
+                         struct sdp_media *m)
```

Comment:
this function should return `int`

if the function is successful, it should return zero.
for any other errors, it should return `errno`.